admin/build.sh

```diff
@@ -31,7 +31,11 @@ if [[ "$CI_TAG" =~ ^v[0-9]+\.[0-9]+(\.[0-9]+)?(-[A-Za-z0-9-]+)? ]]; then
   ls -l *.sbt project/*.scala
 fi
 
-sbt -Dslick.testkit-config=$TESTKIT_CONF "$publishVersion" ++$SCALA_VERSION testAll updateSampleHelloSlick "root/testSample1" "project root" updateSampleSlickPlainsql "root/testSample2" "project root" updateSampleSlickMultidb "root/testSample3"  "project root" updateSampleSlickTestkitExample "root/testSample4" "project root" $extraTarget
+sbt -Dslick.testkit-config=$TESTKIT_CONF \
+  coverage "$publishVersion" ++$SCALA_VERSION testAll coverageReport \
+  updateSampleHelloSlick "root/testSample1" "project root" updateSampleSlickPlainsql \
+  "root/testSample2" "project root" updateSampleSlickMultidb "root/testSample3" \
+  "project root" updateSampleSlickTestkitExample "root/testSample4" "project root" $extraTarget
```

Comment:
Would be nicer to break up the lines more logically (you can apply this suggestion by clicking Commit Suggestion)

```suggestion
sbt -Dslick.testkit-config=$TESTKIT_CONF \
  coverage "$publishVersion" ++$SCALA_VERSION testAll coverageReport \
  updateSampleHelloSlick "root/testSample1" \
  "project root" updateSampleSlickPlainsql "root/testSample2" \
  "project root" updateSampleSlickMultidb "root/testSample3" \
  "project root" updateSampleSlickTestkitExample "root/testSample4" \
  "project root" $extraTarget
```